src/languages/de-DE.json

```
@@ -0,0 +1,278 @@
+{
+	"global_executed_by": "Ausgeführt von {user_name}#{user_discriminator}",
+	"global_missing_permission": ":lock: Fehlende Rechte",
+	"global_missing_permission_bot": "Dem Bot fehlen Berechtigungen um dies durchzuführen! Bitte überprüfe die Rechte für den Bot.\n{missing}",
+	"global_user_not_found": "{emote_warn} Benutzer konnte nicht gefunden werden",
+	"global_channel_not_found": "{emote_warn} Channel konnte nicht gefunden werden",
+	"global_mod_action_log": "{action} by: {moderator_tag} ({moderator_id}) | Target: {target_tag} ({target_id}) | Grund: {reason}",
+	"global_logging_mod": "`[{timestamp}]` {emoji} **{target_tag}** `({user_id})`wurde für {action} von **{moderator_tag}** `({moderator_id})` für **{reason}** `[#{infractionId}]`",
+	"global_logging_unban_auto": "`[{timestamp}]` {emoji} **{target_tag}** `({user_id})`wurde für {action} von **{moderator_tag}** `({moderator_id})` Nachdem die Strafe abgelaufen ist `[#{infractionId}]`",
+	"global_logging_mod_temp": "`[{timestamp}]` {emoji} **{target_tag}** `({user_id})`wurde für {action} von **{moderator_tag}** `({moderator_id})` bis **{until}** für **{reason}** `[#{infractionId}]`",
```

Same as with `global_logging_mod`: ````({user_id})`wurde von **{moderator_tag}** `({moderator_id})` bis **{until}**  {action}, für den folgenden Grund: **{reason}** `[#{infractionId}]````
```suggestion
	"global_logging_mod_temp": "`[{timestamp}]` {emoji} **{target_tag}** `({user_id})`wurde von **{moderator_tag}** `({moderator_id})` bis **{until}** für **{reason}** `[#{infractionId}]` {action}",
```